### PR TITLE
Refactor drilldown code and add SQL_Server set up

### PR DESCRIPTION
**Pull request by marthasharkey**
### Checklist

Please ensure that the following checklist has been satisfied before submitting the PR:

- [ ] The documentation has been updated, if necessary.
- [ ] Screenshots/screencasts have been attached, if there are any visual changes. For interactive or animated visual changes, a screencast is preferred.
- [ ] All code follows the
      [Scala](https://github.com/enso-org/enso/blob/develop/docs/style-guide/scala.md),
      [Java](https://github.com/enso-org/enso/blob/develop/docs/style-guide/java.md),
      [TypeScript](https://github.com/enso-org/enso/blob/develop/docs/style-guide/typescript.md),
      and
      [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md)
      style guides. In case you are using a language not listed above, follow the [Rust](https://github.com/enso-org/enso/blob/develop/docs/style-guide/rust.md) style guide.
- [ ] Unit tests have been written where possible.
- [ ] If meaningful changes were made to logic or tests affecting Enso Cloud integration in the libraries, 
      or the Snowflake database integration, a run of the [Extra Tests](https://github.com/enso-org/enso/actions/workflows/extra-nightly-tests.yml) has been scheduled.
  - If applicable, it is suggested to paste a link to a successful run of the Extra Tests.

### CODE REVIEW

**Review by kazcw — APPROVED** [review]

File: distribution/lib/Standard/Database/0.0.0-dev/src/Connection/Postgres.enso [hunk 1]
```diff
@@ -9,6 +9,7 @@ import project.Connection.Connection_Options.Connection_Options
 import project.Connection.Credentials.Credentials
 import project.Connection.Postgres_Connection.Postgres_Connection
 import project.Connection.SSL_Mode.SSL_Mode
+import project.Internal.Common.Connections_Helpers
```
Comment (radeusgd) [inline distribution/lib/Standard/Database/0.0.0-dev/src/Connection/Postgres.enso]: Why is this import needed?

**Review by radeusgd — APPROVED** [review]

File: distribution/lib/Standard/Database/0.0.0-dev/src/Connection/Postgres.enso (hunk 1 above)
Comment (marthasharkey) [reply distribution/lib/Standard/Database/0.0.0-dev/src/Connection/Postgres.enso]: I have just added this in the wrong place thanks for spotting
